databroker/in_memory.py

```diff
@@ -7,18 +7,10 @@
 from mongoquery import Query
 
 
-from .core import parse_handler_registry, discover_handlers
+from .core import parse_handler_registry, discover_handlers, Entry
 from .v2 import Broker
 
 
-class SafeLocalCatalogEntry(intake.catalog.local.LocalCatalogEntry):
-    # For compat with intake 0.5.1.
```

Comment:
have we pinned to make sure we have a new enough version?

Reply:
Yes, we are pinned to 0.5.2 which (I have just re-confirmed) has the fix.